app/src/main/res/layout/fragment_launch_details.xml

```diff
@@ -14,13 +14,21 @@
             android:orientation="vertical">
 
         <ImageView
-            android:id="@+id/ivLaunchPictures"
+            android:id="@+id/ivDefaultImage"
             android:layout_width="match_parent"
             android:layout_height="300dp"
             android:contentDescription="@string/launch_image_description"
             android:scaleType="fitXY"
+            android:visibility="gone"
             app:srcCompat="@drawable/spacex_logo" />
 
+        <!-- viewpager to show images -->
+        <androidx.viewpager.widget.ViewPager
+            android:id="@+id/vpLaunchPhotos"
+            android:layout_width="match_parent"
+            android:layout_height="300dp">
+        </androidx.viewpager.widget.ViewPager>
+
             <LinearLayout
```

Comment:
@Celina, further down on this xml file, your imageButtons have still have their default names. All of your code is so meticulously done, I really have no comments lol

Reply:
Thanks for the catch! Updated here: [a48d194](https://github.com/onramp-io/twitch-android-project-2/pull/20/commits/a48d194ce83bc6f7975a2b2e23beecaa5208f9a0)